documentation/file_structure.md

```diff
@@ -15,19 +15,30 @@ Directory | Description
 Directory | Description
 :---------|:------------
 /arch	  | Architecture and platform specific files
+/include  | Header files of resources exported in the core
 /lib	  | Generic libraries that are likely to be replaced in a final product
 /mm	  | Generic memory management, currently empty
 /tee	  | Generic TEE files
 
+## Structure of /core/include
+Directory | Description
+:---------|:------------
+/crypto	  | Include files expoising API for /core/crypto files
```

Comment:
s/expoising/exposing/

Not sure this whole file is useful actually. Only a small number of directories really deserve to be described IMO.
Perhaps we should just list the important or non-obvious directories in a later PR.

Reply:
Ok. The ones that deserve a description IMO are **drivers/** and **dt-bidings/**.
**kernel/**, **mm/** and **tee/** provides many APIs. I'll remove them but tell me if there's one worth mentioning.